spec/features/user_browses_by_category_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'any user is able to see' do
+  before(:each) do
+    @category = Category.create(name: "Ghost")
+    @category2 = Category.create(name: "Monster")
+
+    item1 = {
+            title: "Eat Cheese and Die",
+            description: "The spookiest grilled cheese.",
+            price: 50.65
+    }
+
+    item2 = {
+            title: "It Came From Beneath the Sink",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.